src/commands/contract/view_storage/output_format/as_json.rs

```diff
@@ -26,14 +26,19 @@ impl AsJsonContext {
                 if let near_jsonrpc_primitives::types::query::QueryResponseKind::ViewState(result) =
                     query_view_method_response.kind
                 {
-                    eprintln!("Contract state (values):");
-                    println!(
-                        "{}",
-                        serde_json::to_string_pretty(&result.values)?
+                    if let crate::Verbosity::Quiet = previous_context.global_context.verbosity {
+                        println!("Contract state (values):\n{}\n", serde_json::to_string_pretty(&result.values)?);
+                        println!("Contract state (proof):\n{:#?}\n", result.proof)
+                    }
+                    tracing::info!(
```

Comment:
else?

To be honest, I believe that tracing could be configured with using target to display `output` and to remove `noise`

this will help to reduce that repeateness. I would ask you to take a look a bit into that direction. 
I don't mind if it will come in the next PR.

Reply:
Ok